prime-router/src/main/kotlin/serializers/CsvSerializer.kt

```diff
@@ -102,11 +104,21 @@ class CsvSerializer(val metadata: Metadata) {
                 )
             } catch (ex: CSVParseFormatException) {
                 errors.add(
-                    ResultDetail.report(InvalidReportMessage.new("General CSV parsing error on row: ${ex.rowNum}"))
+                    ResultDetail.report(
+                        InvalidReportMessage.new(
+                            "There's an issue parsing your file. Contact the " +
+                                "ReportStream team at reportstream@cdc.gov."
```

Comment:
This email address could change in the future.  I would define this in a common place then reuse it.